src/vconsole/vconsole-setup.c

```diff
@@ -70,29 +70,19 @@ static int verify_vc_allocation_byfd(int fd) {
         return verify_vc_allocation(vcs.v_active);
 }
 
-static int verify_vc_kbmode(int fd) {
-        int curr_mode;
-
-        /*
-         * Make sure we only adjust consoles in K_XLATE or K_UNICODE mode.
-         * Otherwise we would (likely) interfere with X11's processing of the
-         * key events.
-         *
-         * http://lists.freedesktop.org/archives/systemd-devel/2013-February/008573.html
-         */
-
-        if (ioctl(fd, KDGKBMODE, &curr_mode) < 0)
-                return -errno;
-
-        return IN_SET(curr_mode, K_XLATE, K_UNICODE) ? 0 : -EBUSY;
-}
-
 static int toggle_utf8(const char *name, int fd, bool utf8) {
         int r;
         struct termios tc = {};
 
         assert(name);
 
+        r = vt_verify_kbmode(fd);
+        if (r == -EBUSY) {
+                log_debug_errno(r, "Virtual console %s is not in K_XLATE or K_UNICODE: %m", name);
+                return 0;
+        } else if (r < 0)
+                return log_debug_errno(r, "Failed to verify kbdmode on %s: %m", name);
```

Comment:
all other failure codepaths in this functions log loudly, this one should hence too, i.e. bump this one line from log_debug_errno() to log_warning_errno()